packages/graphql/src/schema/get-obj-field-meta.ts

```diff
@@ -91,11 +92,13 @@ function getObjFieldMeta({
             const relationshipMeta = getRelationshipMeta(field);
             const cypherMeta = getCypherMeta(field);
             const typeMeta = getFieldTypeMeta(field);
+            const aliasMeta = getAliasMeta(field);
```

Comment:
Shouldn't this be inside the `if (aliasDirective)` block below? The same could be said for the other "get*Meta" functions above - it feels a bit wasteful to call a function if we _know_ it's going to return undefined.

Reply:
Usually I'd vote for consistency in the code - but sure, I'll move it.